app/src/main/java/org/mozilla/focus/activity/MainMediator.java

```diff
@@ -62,6 +62,11 @@ public void showUrlInput(@Nullable String url) {
         this.prepareUrlInput(url).addToBackStack(UrlInputFragment.FRAGMENT_TAG).commit();
     }
 
+    // If openInNewTab is not provided, we decide based on current state of MainMediator
+    public void showBrowserScreen(final @NonNull String url) {
```

Comment:
@walkingice I and @pharspher were discussing if it is appropriate for the name showBrowserScreen to handle based on isHomeFragmentVisible(). (Can we rename this to make it more explicit that this operation/API is based on state of isHomeFragmentVisible or MainMediator)